news/104.feature

```diff
@@ -0,0 +1 @@
+Provide packing command @erral
```

Comment:
I tried to determine the command's type, but I can't tell if it is a shell, Docker, or make command, and it would be good to clarify that context.

```suggestion
Provide command to pack ZODB. @erral
```